game.libretro.dosbox/addon.xml.in

```diff
@@ -5,6 +5,7 @@
         provider-name="Libretro">
     <requires>
         <import addon="game.libretro" version="1.0.0"/>
+        <import addon="game.controller.mouse" version="1.0.0"/>
```

Comment:
Will do, that needs some changes in core, as I currently just load the first controller type for mouse. (in OpenMouse)